book/invalidation.md

```diff
@@ -780,9 +818,17 @@ invalidation scenarios much easier, and it will push us toward making
 every field protected.
 
 ::: {.further}
-
+A classic bug in all dirty-bit-based systems is [under-invalidation],
```

Comment:
This sentence repeats the other go-further..

Reply:
Uh oh, probably an error when I was moving around the go-furthers between sections. will fix.